<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mine</class>
 <widget class="QWidget" name="Mine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>445</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>我的信息</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>地址</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>手机号</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QLineEdit" name="msg6">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>邮箱</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLineEdit" name="msg1">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="placeholderText">
        <string>不能为空</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QTextEdit" name="msg7">
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLineEdit" name="msg2">
       <property name="echoMode">
        <enum>QLineEdit::Normal</enum>
       </property>
       <property name="placeholderText">
        <string>不能为空</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLineEdit" name="msg3">
       <property name="placeholderText">
        <string>不能为空</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>性别</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QLineEdit" name="msg5">
       <property name="inputMask">
        <string/>
       </property>
       <property name="placeholderText">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>用户名</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>真实姓名</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QComboBox" name="msg4"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="Sure">
       <property name="text">
        <string>确认修改</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="fresh">
       <property name="text">
        <string>初始化</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="exit">
       <property name="text">
        <string>退出</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>msg1</tabstop>
  <tabstop>msg2</tabstop>
  <tabstop>msg3</tabstop>
  <tabstop>msg5</tabstop>
  <tabstop>msg6</tabstop>
  <tabstop>msg7</tabstop>
  <tabstop>Sure</tabstop>
  <tabstop>fresh</tabstop>
  <tabstop>exit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
